core/blockchain.go

```diff
@@ -1136,6 +1136,7 @@ func (bc *BlockChain) InsertChain(chain types.Blocks) (int, error) {
 	n, events, logs, err := bc.insertChain(chain)
 	bc.PostChainEvents(events, logs)
 	if err == nil {
+		// TODO: incorporate these into insertChain
```

Comment:
any impact? can you open a ticket on this?

Reply:
the TODO is already done in master. But the code at cherrypick doesn't